composer.json

```diff
@@ -46,7 +46,7 @@
         "php-cs-fixer/accessible-object": "^1.1",
         "php-cs-fixer/phpunit-constraint-isidenticalstring": "^1.4",
         "php-cs-fixer/phpunit-constraint-xmlmatchesxsd": "^1.4",
-        "phpunit/phpunit": "^9.6",
+        "phpunit/phpunit": "^9.6 || ^10.5",
```

Comment:
https://github.com/facile-it/paraunit/issues/223

Reply:
@keradus https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7611?

Reply:
solution described in lined paraunit issue